.github/workflows/cmake-multi-platform.yml

```
@@ -0,0 +1,172 @@
+# Github CI config
+# Inspired by https://github.com/libcpr/cpr/blob/master/.github/workflows/ci.yml
+# Runs: clang format check, g++ & clang, release & debug, on Fedora
+
+name: OSTree-TUI CI
+
+on: [push, workflow_dispatch, pull_request]
+
+env:
+  # standard dependencies for building & project
+  BUILD_DEPENDENCIES: "gcc-c++ clang git gcc gdb make cmake automake autoconf"
+  OSTREE_TUI_DEPENDENCIES: "glib2-devel gtk-doc ostree-devel libtool bison liblzf e2fsprogs-devel xz-devel gpgme-devel fuse-devel"
+
+jobs:
+  fedora-clang:
+    strategy:
+      matrix:
+        container: ["fedora:latest"]
+        buildType: [Debug, Release]
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
```

```suggestion
        buildType: [Debug, Release]
    runs-on: ubuntu-latest
    container: fedora:latest
```

Since we only target `fedora:latest` anyway right now. This reduces complexity and the job name `fedora-*` matches.

---

I kept it like this, just in case other targets would be added later on. But I guess it makes sense to only add this complexity, whenever that is needed. Will be integrated :+1: